src/data/businessCase.ts

```diff
@@ -80,9 +73,27 @@ export const prepareBusinessCaseForApp = (
   };
 
   businessCase.lifecycleCostLines.forEach((line: any) => {
-    lifecycleCostLines[line.solution as keyof lifecycleCostLinesType][
-      yearMap[line.year]
-    ].push({ phase: line.phase, cost: line.cost });
+    const solution = (solutionName => {
+      switch (solutionName) {
+        case 'As Is':
+          return lifecycleCostLines['As Is'];
+        case 'Preferred':
+          return lifecycleCostLines.Preferred;
+        case 'A':
+          return lifecycleCostLines.A;
+        case 'B':
+          return lifecycleCostLines.B;
+        default:
+          return null;
+      }
+    })(line.solution);
+
+    if (solution) {
+      solution[`year${line.year}` as keyof EstimatedLifecycleCostLines].push({
+        phase: line.phase,
```

Comment:
Actually is there anything else we need to include in here for the backend? Should we include the `id`?

Reply:
if we have it, yes.